### PR TITLE
Update/bourbon neat

### PR DESCRIPTION
**Pull request by ajrobertsonio**
Updates Bourbon and Neat. 

Please review, @yulianglukhenko, @ryanlntn

### CODE REVIEW

**Comment by silasjmatson** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -3,22 +3,22 @@ gem 'rake'
...
 
 # Deployment Solution
-gem 'middleman-deploy', '~> 0.3'
+gem 'middleman-deploy', '~> 1.0.0'
 
 # Testing
-gem 'capybara', '~> 2.2'
-gem 'rspec', '~> 3.0'
+gem 'capybara', '~> 2.4.4'
+gem 'rspec', '~> 3.2.0'
```
Comment (ryanlntn) [inline Gemfile]: @ajrobertson I don't think we want to version lock this pessimistically. Can you revise to keep the same number of decimal places (usually 2) for each gem? Let me know if you have any questions about that.

**Comment by yulianglukhenko** [comment]
@ajrobertson we dont need to update bourbon/neat. When you pull the repo while setting up the project and run the setup script, it will update it for you. Just btw.

**Comment by ajrobertson** [comment]
Oh, I didn't realize. I'll probably just delete this PR and branch then.

**Comment by yulianglukhenko** [comment]
@ajrobertson yea those are updated running the script. The only thing you have to update manually are the bitters files in the "base". However, there's some custom stuff in our base so don't overwrite them.

**Pull request closed without merging**